game/scripts/vscripts/abilities/twin/boss_twin_twin_empathy.lua

```diff
@@ -0,0 +1,11 @@
+LinkLuaModifier("modifier_boss_twin_twin_empathy", "abilities/twin/modifier_boss_twin_twin_empathy.lua", LUA_MODIFIER_MOTION_NONE)
+
+boss_twin_twin_empathy = class({})
+
+function boss_twin_twin_empathy:GetIntrinsicModifierName()
+  return "modifier_boss_twin_twin_empathy"
```

Comment:
This should be removed since we want just 1 instance of the modifier, and for it to have proper Parent/Caster setup.